de.metas.migration/de.metas.migration.cli/RolloutMigrate.launch

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>

```

Comment:
Thanks for providing this launch config. In order to be useful for others, I think some further information alongside this config is needed in addition:
* which variables to change, e.g. if one's `PG_HOME` is somewhere else. Maybe for that varaible, it's even better to include the recommendation to set `PG_HOME` as a system environment variable.
* a notice that the `-s` argument needs to point to the directory containing the SQL scripts, probably best together with a warning, that an SQL should be in the same module as its respective code.
* a note about the `migrate.properties` file
* maybe more information which you are aware of but doesn't come to my mind right now..


I recommentd to put all this into a markdown file such as `de.metas.migration/de.metas.migration.cli\README_rolloutmigrate.md` so that each prospect contributor who wants to use your launch config can do so without further research..